helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/StringConverterUtil.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+package com.linkedin.avroutil1.compatibility;
+
+import java.nio.charset.StandardCharsets;
+import org.apache.avro.util.Utf8;
+
+
+public class StringConverterUtil {
```

Comment:
Where is this used?

Reply:
i believe the intent is to call this from generated code if/when converting between Utf8 fields and String getters/setters is needed ?

Reply:
This is being written inside generated code for conversion between Charsequence types (String, CharSequence and Utf8)
Converting it to a Util with static methods in the next commit

Reply:
Ah I see. Then similar to the comment above, it could be good to add this note to the javadoc or as a normal comment